### ISSUE TITLE
[Suggestion] More color blending options

### ISSUE THREAD

**Issue by UsernameGeri** [fishcute/Celestial#13]
**Feature**
Being able to use different color blend modes, as seen here: https://en.wikipedia.org/wiki/Blend_modes
From my experience, the most useful ones would be `normal`, `add`, `screen`, `multiply` and `overlay`.

**Potential uses**
This would mostly be useful for more advanced techniques, such as masking certain sky elements, or getting a specific color output.
For example, the `screen` mode gives much softer blending, compared to the blending the vanilla game offers.

**Details**
These new modes could replace/rearrange some of the currently implemented `properties` entries.

For example, the currently undocumented `blend` property could be re-used as a new _Blend Entry_ containing one, from the list of valid blend types.
The `is_solid` property in combinations with `blend` produces the same result as the `normal` blend mode, when both are set to `true`, which would make `is_solid` obsolete.
The currently used blending mode could be named as `vanilla`.

In short, `is_solid` could be removed, and `blend` would use a format like this:
`"blend": "vanilla"`
`"blend": "screen"`
